### PR TITLE
Add a 'docs-doc2dash' Tox target.

### PR DESCRIPTION
**Pull request by tseaver**
Generates a Dash-style docset from the Sphinx-rendered HTML using [doc2dash](http://doc2dash.readthedocs.org/en/stable/).

I've copied the docset manually into my [Zeal](https://zealdocs.org/) library, and it looks pretty good.

Follow-up to [my comment on #1690](https://github.com/GoogleCloudPlatform/gcloud-python/pull/1690/#discussion_r58872605).

### CODE REVIEW

**Comment by dhermes** [comment]
Do we actually need this?

**Comment by jgeewax** [comment]
Yea - confused. What's the goal ?

**Comment by tseaver** [comment]
The goal is to generate [Dash](https://kapeli.com/dash)-compatible docsets, to allow developers to import the docs into their offline API docs (searchable/readable via Dash, [Zeal](https://zealdocs.org/), etc.):  I was trying to address one of the problems with just expecting people to read the online docs.

**Comment by daspecster** [comment]
I know a decent number of people that use Dash.  If this works as is and is low maintenance, I vote to use it.

**Comment by dhermes** [comment]
@tseaver Shall we close this PR?

**Comment by tseaver** [comment]
I've rebased it to resolve the conflict in `.gitignore`.

If we automated building the Dash docsets and [pushing them to Dash](https://github.com/Kapeli/Dash-User-Contributions#contribute-a-new-docset) as part of our tag-release deployment, that seems like a low-cost win to me:  users who like using offline docs will be able to consume them with their preferred tool.

File: tox.ini [hunk 1]
```diff
@@ -79,6 +79,19 @@ deps =
     sphinx_rtd_theme
 passenv = {[testenv:system-tests]passenv} SPHINX_RELEASE READTHEDOCS
 
+[testenv:docs-doc2dash]
+setenv =
+    PYTHONPATH = {toxinidir}/_testing
+basepython = {[testenv:docs]basepython}
+commands =
+    sphinx-build -W -b html -d docs/_build_doc2dash/doctrees docs docs/_build_doc2dash/html
```
Comment (dhermes) [inline tox.ini]: Can you backslash continue these lines?

**Comment by dhermes** [comment]
@tseaver Feel free to merge when my request is addressed.

Are you going to automate the process of creating and publishing this target?

**Comment by tseaver** [comment]
> Are you going to automate the process of creating and publishing this target?

I'm thinking about it.  There seem to be two options:

- [Host the docs ourselves](https://kapeli.com/docsets#dashdocsetfeed), e.g. in the `gh-pages` branch.
- [Create a pull request to the `Dash-User-Contrib` repository](https://github.com/Kapeli/Dash-User-Contributions#contribute-a-new-docset), creating or updating a docset there.

Automating the first seems quite feasible; the second seems infeasible.

**Pull request merged**